packages/terra-date-time-picker/src/_TimeClarification.jsx

```diff
@@ -55,6 +55,10 @@ const propTypes = {
    * Whether the clarification is disabled.
    */
   disabled: PropTypes.bool,
+  /**
+   * Time Zone the date time picker is rendered in.
```

Comment:
Could we pull forward the prop description used in the DateTimePicker file? I think the info on the expected format would be helpful: https://github.com/cerner/terra-framework/pull/1044/files#diff-fda357981bfc2d7e9aebab8b200dc00eR123-R126

Reply:
changed in this commit b5f07bc5f04b127f3d75bc46a3651a2e1091b165